updater/scripts/update-changelog.ps1

```diff
@@ -23,12 +23,32 @@ Write-Host "Found changelog: $file"
 
 [string[]]$lines = Get-Content $file
 
+$skipPrettier = $false
```

Comment:
Can we rename this to `skippingComment` or sth similar? This currently sounds like a configuration instead of a state.

Reply:
skippingComment sounds better for you?

Reply:
indeed it does :D